classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php

```diff
@@ -180,18 +180,16 @@ protected function checkLocaleConflicts(): void
                         fn (Builder $query) => $query->groupBy('setting_name')
                     )
                     ->havingRaw('COUNT(*) >= 2')
-                    ->get();
-
-                if (!$conflictingSettings->isEmpty()) {
-                    foreach ($conflictingSettings as $conflictingSetting) {
-                        $settingsExceptionMessage .= 'A row with "' . $entityIdColumnName . '"="' . $conflictingSetting->{$entityIdColumnName} . '" and "setting_name"="' . $conflictingSetting->setting_name . '" found in table "' . $tableName . '" which will conflict with other rows specific to the locale key "' . $localeTarget . '" after the migration. Please review this row before upgrading.' . PHP_EOL;
```

Comment:
Besides downloading all records, this exception message can become potentially huge.